test/services/markdown_service_test.rb

```diff
@@ -9,6 +9,14 @@ class MarkdownServiceTest < ActiveSupport::TestCase
     assert_equal(output, expected)
   end
 
+  test 'it parses emoji' do
```

Comment:
I just set up houndci today so I have to send in a PR to make the rest of lecture-hall follow it's conventions. Could you make your PR follow it?